src/EFCore.PG/Metadata/PostgresExtension.cs

```diff
@@ -173,7 +173,7 @@ void SetData([CanBeNull] string version)
                 return (null, null, null);
 
             // TODO: Can't actually use schema and name...they might not be set when this is first called.
-            var schemaNameValue = value.Split(',').Select(x => x == "" ? null : x).ToArray();
+            var schemaNameValue = value.Split(',').Select(x => x.Trim()).Select(x => x == "" || x == "''" ? null : x).ToArray();
```

Comment:
Am curious about the `''` case - where/how did you encounter this?

Reply:
Please look at related issue #899, this case explained there.

Reply:
Oh sorry, missed that - will comment there.